package.json

```diff
@@ -1,22 +1,18 @@
 {
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2022-2421   
**Check Name:** Insufficient validation when decoding a Socket.IO packet   
**Severity:** CRITICAL   
**Fixed Version:** 4.2.1, 4.0.5   
**Description:** Due to improper type validation in attachment parsing the Socket.io js library, it is possible to overwrite the _placeholder object which allows an attacker to place references to functions at arbitrary places in the resulting query object.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2022-2421